prov/util/src/util_attr.c

```diff
@@ -706,11 +706,17 @@ int ofi_check_info(const struct util_prov *util_prov, uint32_t api_version,
 		return -FI_ENODATA;
 	}
 
-	if (FI_VERSION_LT(api_version, FI_VERSION(1, 5)))
+	/* If the app sets FI_MR_LOCAL, we ignore FI_LOCAL_MR.  So, if the
+	 * app doesn't set FI_MR_LOCAL, we need to check for FI_LOCAL_MR.
+	 * The provider is assumed only to set FI_MR_LOCAL correctly.
+	 */
+	if (FI_VERSION_LT(api_version, FI_VERSION(1, 5)) ||
```

Comment:
Sorry, I'm not sure that it's right. But I'm asking about handling of mode bits in `ofi_check_tx/rx_attr` function. Should we add the same handling in that function?